src/registrar/templates/domain_request_form.html

```diff
@@ -130,9 +130,21 @@ <h1> {{form_titles|get_item:steps.current}} </h1>
           aria-describedby="Are you sure you want to submit a domain request?"
           data-force-action
         >
-            {% include 'includes/modal.html' with is_domain_request_form=True review_form_is_complete=review_form_is_complete modal_heading=modal_heading|safe modal_description=modal_description|safe modal_button=modal_button|safe %}
+          {% if review_form_is_complete %}
+            {% with modal_heading="You are about to submit a domain request for " domain_name_modal=requested_domain__name modal_description="Once you submit this request, you won’t be able to edit it until we review it. You’ll only be able to withdraw your request." modal_button_id="domain-request-form-submit-button" modal_button_text="Submit request" %}
```

Comment:
Q: Why the nested with structure instead of the more straightforward attributes on the {% include %}

Reply:
fixed, nice catch